### PR TITLE
remove incorrect reference to `redirect_uri`

### LINKED ISSUE #34651: remove incorrect reference to `redirect_uri`
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-a-user-access-token-for-a-github-app

### What part(s) of the article would you like to see updated?

`redirect_uri` is **not** a parameter to the call that exchanges the OAuth2 code for the token. It's a parameter to an earlier call: the one that redirects the user back to the app.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by akorzynski**
### Why:

`redirect_uri` is **not** a parameter to the call that exchanges the OAuth2 code for the token. It's a parameter to an earlier call: the one that redirects the user back to the app.

Closes: #34651 

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Removed the `redirect_uri` reference from the table containing the query parameters.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@akorzynski Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by nguyenalex836** [comment]
@akorzynski Thank you for your patience while our SME team reviewed your PR! 

After review, they responded with the following to whether `redirect_uri` is a parameter to the call that exchanges the OAuth2 code for the token -

> Per RFC 6749 section 4.1.3 this is a valid use of the parameter. It is optional, as implied by the lack of a Required note. If included though it must match.

For this reason, we will close this PR 💛 Thank you for your willingness to contribute to our docs! ✨

File: data/reusables/apps/web-app-flow-exchange-code.md [hunk 1]
```diff
@@ -5,5 +5,4 @@
    `client_id` | `string` | **Required.** The client ID for your {% data variables.product.prodname_github_app %}. The client ID is different from the app ID. You can find the client ID on the settings page for your app. For more information about navigating to the settings page for your {% data variables.product.prodname_github_app %}, see "[AUTOTITLE](/apps/maintaining-github-apps/modifying-a-github-app-registration#navigating-to-your-github-app-settings)."
```
Comment (midojms-del) [inline data/reusables/apps/web-app-flow-exchange-code.md]: a2f178351afc3ad570c5089a9deec80bc9e5f0e7

Comment (alihussnian052-gif) [inline data/reusables/apps/web-app-flow-exchange-code.md]: Dependencies installed


✅ Development server starts (usually http://localhost:3000)


✅ Hot-reload enabled for live updates


✅ Ready to make code changes

**Review by bigbob78C — COMMENTED** [review]
The threats are always there

**Pull request closed without merging**